docs/proposals/PROPOSAL_D.md

```diff
@@ -0,0 +1,480 @@
+
+## Requirements
+
+Taken from [REQUIREMENTS](../REQUIREMENTS.md). Points that have:
+
+- `TOOL RESPONSIBILITY` would be possible by a custom tool.
+- `AGNOSTIC`: the proposal is agnostic to the specific point (can be decided by working group)
+
+### Image Author
+
+- [x] As an image author, I want to update compatibility independently without having to re-release and re-distribute my image.
+- [x] As an image author, I want to have the freedom to express any compatibility that is necessary for my container to run on the host.
+  - [x] Including conditional compatibility, such as a container running on AMD CPU and intel CPU having different requirements.
+  - [x] Including must have/nice to have compatibility.
+  - [x]  Matching finer grain platform definitions.
+- [x] As an image author, I want to use a provided tool to verify the compatibility spec I wrote against the schema.
+- [ ] As an image author, I would like to create a single compatibility description that is common to a group of images (`TOOL RESPONSIBILITY`).
+- [ ] As an image author, I want to be able to specify in my manifest that my multi-platform image has a compatibility spec which should be consulted `AGNOSTIC`.
+- [ ]  As an image author, I want to be able to include compatibility specifications from base layers that I inherited from `TOOL RESPONSIBILITY` that can derive path of base layers
+- [x] As an image author, I want to ensure that runtimes without image compatibility gracefully fall back to running a usable image.
+
+### Domain Architect
+
+- [x] As a domain architect I want a process to share my knowledge about \<niche topic\> compatibility with some compatibility interest group.
+- [x] As a domain architect I don't want to have to understand containers or develop tools for them to share this knowledge.
+
+### Tool Writer
+
+- [x] As a tool writer, I want to get the compatibility spec without pulling the image layer blobs.
+- [x] As a tool writer, I would like to have a library for reading image compatibility so that I can write my own software that takes action based on the spec, e.g.:
+  - [x] custom k8s scheduler, admission webhooks, runtime classes etc.
+- [x] As a tool writer, compatibility validation could be integrated into non-runtime tools.
+- [x] As a tool writer, I want to be able to write tools (that use compatibility specs) that have non-standard applications (e.g., checking individual layers).
+
+### System Runtime Administrator
+
+- [x] As a system runtime administrator, I want to check whether a container is compatible with the nodes I am going to run it on using the provided tool.
+- [x] As a system runtime administrator, I would like to fetch additional documentation for understanding specific settings in the compatibility spec.
+- [x] As a system runtime administrator, selecting which image to run should only require pulling the Index manifest, and parsing the descriptors listed.
```

Comment:
I believe this one is mutually exclusive with line 407.

Reply:
I just copy pasted the contents from the requirements file.

Reply:
This was a reference to checking the item. If we say "I want to update compatibility independently without having to re-release and re-distribute my image" is provided by this implementation, then I don't think we can also say a runtime can select an image with only the Index manifest. Runtimes would need to pull the associated referrers to support that.

Reply:
I checked it because you still technically could - it would work as it does now.

Reply:
@sudo-bmitch I removed this box, because the implication is "I want to get compatibility information _only_ using the index" and not "I want to still be able to select an image" (how I read it).